docs/learn/learn-comparison-rollups.md

```diff
@@ -0,0 +1,155 @@
+---
+id: learn-comparisons-rollups
+title: Layer Two and Rollups
+sidebar_label: Layer Two and Rollups
+description:
+  Comparisons between various rollup and layer two protocols and how they relate to Polkadot.
+keywords: [rollups, polkadot, scalability, shared, security, parachain, ethereum]
+slug: ../learn-comparisons-rollups
+---
+
+:::note
+
+This comparison covers general information regarding two widely used rollup mechanisms that are used
+to scale (usually EVM-based) blockchains and compares and contrasts how Polkadot deals with
+scalability.
+
+:::
+
+Layer two networks are notorious as being the way forward for blockchain scalability by off-loading
+the majority of computation off-chain. Often, layer two solutions are needed to scale an existing
+layer one blockchain. They utilize the layer one network's security and functionality to build an
+additional layer that is often faster, reduces fees, and solves other platform-specific issues. In
+many cases, layer two solutions focus on capitalizing the use of blockspace within a particular
+blockchain to make it less costly and more efficient.
+
+**Rollups** are a layer-2 scaling solution. At the most basic level, a layer two solution is
+responsible for "rolling up" transactions by batching them before publishing them to the layer one
+chain, usually through a network of **sequencers**. This mechanism could include thousands of
+transactions in a single rollup.
+
+Polkadot implements this functionality at the native level, allowing for shared security and
+scalability of the relay chain and respective parachains. Shared security is a concept that has
+similar goals to EVM-based optimistic and zero-knowledge rollups. Still, instead of being
+implemented as a secondary layer, Polkadot guarantees native security and scalability for each of
+its parachains through the [Parachains Protocol](./learn-parachains-protocol.md). Polkadot handles
+the coordination of data from parachains into an aggregated, representative state, somewhat similar
+to how to layer two rollups function.
```

Comment:
```suggestion
**Rollups** are a L2 scaling solution. At the most basic level, a rollup L2 solution is
responsible for "rolling up" transactions by batching them before publishing them to the L1
chain, usually through a network of **sequencers**. This mechanism could include thousands of
transactions in a single rollup.

{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements this functionality at the native level (i.e. without using L2 scaling solutions), allowing for shared security and
scalability of the relay chain and respective parachains. Shared security is a concept that has
similar goals to EVM-based optimistic and zero-knowledge rollups. Still, instead of being
implemented as a secondary layer, {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} guarantees native security and scalability for each of
its parachains through the [Parachains Protocol](./learn-parachains-protocol.md). {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} handles
the coordination of data from parachains into an aggregated, representative state, somewhat similar
to L2 rollups.
```